installer/templates/static/brunch/brunch-config.js

```diff
@@ -51,6 +51,13 @@ exports.config = {
       ignore: [/web\/static\/vendor/]
     }
   },
+
+  modules: {
+    autoRequire: {
+      'js/app.js': ['app']
```

Comment:
Have you tried this out? Because I think the require should be `web/static/js/app`.